to-do-list/src/API/PostService.tsx

```diff
@@ -0,0 +1,39 @@
+import axios from 'axios'
+import {PostItemProps} from '../components/Interfaces/interfaces'
+
+export default class PostService {
+    static async getAll(){
```

Comment:
Don't forget about the typescript. You need to specify all the returning values of the functions